packages/engine/src/store/actionCreators/createUpdateInputValues.ts

```diff
@@ -0,0 +1,16 @@
+import { NodeValue, SetterCreator } from '@store/types'
+
+export const createUpdateInputValues: SetterCreator<'updateInputValues'> =
```

Comment:
I'm not totally sure if this should even be a store action, as we're mutating state. I realise this exactly how i'd done it for updating values with `updateNodeValue` but now I'm having a rethink. No need to change in this PR, more of a note for myself! I will address this later.